class HatePythonError(Exception):
    pass

class Person:
    def __init__(self, name, age):
        self.name = name
        self.age = age
    
    def greeting(self):
        print("Hello, My name is", self.name)
        if str(self.name).lower() == "alex4386":
            print("Wait.... You put me in the python?")



            print("F̴̨̳̠͇̬͉̪̜̰̻̦̹̮͓̈́̕Ų̶̧̝͉͎̻̜͚̝͋͆̃̈́̕̕U̷̳͎͖͉̻͔͖̘̣̳͘͜Ǘ̶̢͎͕̲̳̬̼͖̘̙̆̒̃́̓̌͊U̴̢̧̘̖̞̬̩̯͔̫͗̊̍̂̐̓͐̾́̈̋̆̓̑͠ͅU̶̜̜̺̗̗̩̤͆̈̀̐͠Ų̷̧̻͎̯̮͙͉̹̗͇̰̳͗͂̂͆̾̄̊͒̆̓̎̐ͅU̶͎̬̟̗̹̗̓͛̓̾̓̔͆̇̑̇̃̒͘ͅƯ̷̜̗̻̼̩͂̑̄̎̑̓̒̚͜͜͠͝")



            
            raise HatePythonError("F̶̨̧̣̖̖̥̟̗̟̱̗̭͈̥̦̙̖̹͎̯̣̙̔͊̈̀̈́͛͂͛̆̎͊̀̃̓̂͗̌͛͂̃̐̄̋̔̅̓̇̚̚͝Ư̴͓̻̱͉̞̪̣͍͎̗͙̓̀̔͑̉͗̆̿̍̒̅̂̋̅̾́̾̿̄̊́̐͘͘͝͠͠C̸̢̡̡̨̡̛̛̩̻͕̝͕͉͚̞̥͓̙̞̲̺̱̘̝̙̰̺̞̣̻̣̞̣̰̮͑̽̀̔̐̂̄̇̿̊̂͌̋̈́͆̍̂͆̍̍̈́͂͐́̿̓̈́̈́̏͐̽̍̿̂͊͛̎̍͘̕̚͜͝͝Ķ̴̛͚̹̙͈͔̫̤͍͚͈̲̘͚̳̻̼̥̲̘͍͙̩̘̲̹̫̰̬̞̟͌̍̈́͊̏͒̋̔̓͐̅̏͋͛̂́͛̂͑́͗̋̑̓̀̐͋͐̇̌̇̿̂̚̕͠ ̶̥͇̜̤̻̼̣̺̥̭̯̻͉͔̠̮̩͙͕̤̝͎̻̘͕̠͑̃̎̽͊̄͂̈͋̔͒̅͂͂̂̌̂̅̾͋̌͗̀̾̑́̓̔̑̽̕͘͝͝͝Y̶̧̡̧̖̫̙̬͝O̷̡̜̰̭̬͔̩͕͙̣͖̣̰̩͔͉̤̝̭͍͂̓͛̑̌̐̓͗͊U̵̢̳̞̙̲̞͙̘̱̰̙̮̼̼̟͚̺̝̖̺̺̖̣̰̖̥̜̘͙͇̙̳̘͚͍̱̲̻͐͊̑͐̎̃̽̐̒͊̍̉̇̐̊̽̈́͐̍̀͑̊̇̍̾̉͗͒̔̾̑͗")


john = Person("John", 36)
john.greeting()

alex4386 = Person("Alex4386", 18)
alex4386.greeting()
